src/components/sections/hero/hero.component.tsx

```diff
@@ -1,13 +1,13 @@
 import Image from "next/image";
-import heroImg from "public/static/hero-logo-img.png";
+import heroImg from "public/images/hero-logo-img.png";
```

Comment:
I found module errors while I'm running Storybook, please check the Storybook view to check and refine them.

Reply:
@tridenda You need to be more specific about the error. Can you share the screenshot of the same and also make sure you are using the new snapshot which has storybook-addon-next installed. 

or May be you have improperly merged the code.